Hunting/Assets/Scripts/AnimalBehaviour/AnimalBehaviour.cs

```diff
@@ -184,12 +189,23 @@ protected void Attack(bool isPlayer)
         if (isPlayer)
         {
             PlayerHealth playerHealth = reach.playerInRange.GetComponentInChildren<PlayerHealth>();

```

Comment:
Not sure how much we care about code style, but it'd make more sense if the `PlayerHealth` is updated via the `PlayerController` instead of directly accessing it from inside `AnimalBehavior`

Reply:
:ok_hand:

Reply:
After looking at the code and assets again, I'm not very sure if this is a good idea (even though this is the better practice) for our case. 
I wrote PlayerController to control our temporary player game object which is independent of VR, because I don't really know how to run the game without VR, without Unity crashing. 
There is another instance of PlayerHealth being used: in the Hierarchy, it is a component of [VRTK_Scripts]/PlayArea.
I am worried that if I modify PlayerHealth to depend on PlayerController, it will break the other VR dependencies.

Reply:
I'll take a look

Reply:
I can split PlayerController into PlayerMovement and PlayerLook, which are intuitively independent from PlayerHealth (though it is still best if each of these classes's state are mutated by an overarching PlayerController).